neutromeratio/tautomers.py

```diff
@@ -174,7 +174,7 @@ def _return_energy(mol, energy_function):
                 pos = mol.GetConformer(0).GetAtomPosition(a.GetIdx())
                 tmp_coord_list.append([pos.x, pos.y, pos.z])
             x = np.array(tmp_coord_list) * unit.angstrom
-            e, _, stddev, ___ = energy_function.calculate_energy(x)
+            e, _, stddev, ___, ____ = energy_function.calculate_energy(x)
```

Comment:
haven't seen this before: does `x, _, __, ___ = ...` have different behavior from `x, _, _, _ = ...`?

Reply:
not at all -- but that's an error anyway, since `calculate_energy()` returns now a NamedTuple! Will change this!